CHANGELOG.md

```diff
@@ -1,7 +1,53 @@
 # Changelog
 
-## [v1.15.0-rc](https://github.com/skywinder/github-changelog-generator/tree/v1.15.0-rc) (2017-10-29)
-[Full Changelog](https://github.com/skywinder/github-changelog-generator/compare/v1.15.0.pre.beta...v1.15.0-rc)
+## [v2.0.0](https://github.com/skywinder/github-changelog-generator/tree/v2.0.0) (2018-04-12)
+
+[Full Changelog](https://github.com/skywinder/github-changelog-generator/compare/v1.15.0.pre.rc...v2.0.0)
```

Comment:
noticed all URLs still use "skywinder"; that something to be changed?

Reply:
Hello!

(And yes, we can change those.)